src/main/java/use_case/TodoListInfoFromTaskReaders.java

```diff
@@ -0,0 +1,74 @@
+package main.java.use_case;
+
+import main.java.entity_gateway.TaskReader;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TodoListInfoFromTaskReaders implements TodoListsInfo {
```

Comment:
This class is immutable and a new one will be created every time a request of todoist info arrives. Is this class for one-time use? If so, should we remove it from memory after using?

Reply:
Yes DTOs are typically for one-time use (at least from what I know). Java's garbage collection should automatically trash this as it's not allocating any external resources.